src/test/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleStitchingServiceTest.java

```diff
@@ -108,4 +109,34 @@ public void testUpdateCaseMissingBundles() throws Exception {
         assertNull(node.get(0));
     }
 
+    @Test(expected = InputValidationException.class)
+    public void testUpdateCaseFileNameOneChar() throws Exception {
+        CcdCallbackDto ccdCallbackDto = new CcdCallbackDto();
+        JsonNode node = objectMapper.readTree("{\"cb\":[{\"value\":{\"eligibleForStitching\":\"yes\", "
+            + "\"fileName\":\"a\"}}]}");
+        ccdCallbackDto.setPropertyName(Optional.of("cb"));
+        ccdCallbackDto.setCaseData(node);
+        ccdCallbackDto.setJwt("jwt");
+
+        ccdBundleStitchingService.updateCase(ccdCallbackDto);
+
+    }
+
```

Comment:
why the two different approaches to testing the exceptions, seems like the second approach allows more granular testing